src/safetywrap/_impl.py

```diff
@@ -176,6 +176,18 @@ def expect(self, msg: str, exc_cls: t.Type[Exception] = RuntimeError) -> T:
         """
         return self._value
 
+    def raise_if_err(
+        self, msg: str, exc_cls: t.Type[Exception] = RuntimeError
+    ) -> T:
+        """Return `Ok` value or raise an error with the specified message.
+
+        The raised exception class may be specified with the `exc_cls`
+        keyword argument.
+
+        Alias for `Ok.expect`.
+        """
+        return self.expect(msg, exc_cls)
```

Comment:
Could we provide `exc_cls` as a keyword argument here, so that it's not dependent on the ordering of available kwargs, just in case we add one in the future?

```suggestion
        return self.expect(msg, exc_cls=exc_cls)
```

Reply:
sure. and will do so elsewhere as well.